activerecord/test/cases/finder_test.rb

```diff
@@ -1406,7 +1406,7 @@ def test_condition_utc_time_interpolation_with_default_timezone_local
     with_env_tz "America/New_York" do
       with_timezone_config default: :local do
         topic = Topic.first
-        assert_equal topic, Topic.where(["written_on = ?", topic.written_on.getutc]).first
+        assert_equal topic, Topic.where(["written_on = ?", topic.written_on]).first
```

Comment:
This test is no longer testing what it says it is.

By my reading, it looks like this behaviour has been broken (for MySQL + prepared statements + local-vs-UTC time values) since 8ebe1f2feed